flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/sql/ScalarSqlFunctions.scala

```diff
@@ -58,4 +58,22 @@ object ScalarSqlFunctions {
       OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.NUMERIC)),
     SqlFunctionCategory.NUMERIC)
 
+  val SHIFT_LEFT = new SqlFunction(
+    "SHIFT_LEFT",
+    SqlKind.OTHER_FUNCTION,
+    ReturnTypes.ARG0,
+    null,
+    OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.NUMERIC),
```

Comment:
Second operand should be  SqlTypeFamily.INTEGER.
 `OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.NUMERIC) -> OperandTypes.family(SqlTypeFamily.NUMERIC, SqlTypeFamily.INTEGER) `